src/spikeinterface/core/analyzer_extension_core.py

```diff
@@ -520,6 +520,34 @@ def _select_extension_data(self, unit_ids):
 
         return new_data
 
+    def get_templates(self, unit_ids=None, save=True):
+        """
+        Return average templates for multiple units.
+
+        Parameters
+        ----------
+        unit_ids: list or None
+            Unit ids to retrieve waveforms for
+        save: bool, default True
+            In case, the operator is not computed yet it can be saved to folder or zarr.
+
+        Returns
+        -------
+        templates: np.array
+            The returned templates (num_units, num_samples, num_channels)
+        """
+
+        templates = self.data["average"]
+
+        if save:
+            self.save()
```

Comment:
This is strange beacause "average" is already in templates.
It was the case in TemplateExtension because operator where computed on the fly.

Reply:
Indeed, this should be removed!